src/libs/EmojiTrie.js

```diff
@@ -18,26 +18,40 @@ function createTrie(lang = CONST.LOCALES.DEFAULT) {
             return;
         }
 
-        const name = isDefaultLocale ? item.name : _.get(langEmojis, [item.code, 'name']);
-        const names = isDefaultLocale ? [name] : [...new Set([name, item.name])];
-        _.forEach(names, (nm) => {
-            const node = trie.search(nm);
-            if (!node) {
-                trie.add(nm, {code: item.code, types: item.types, name: nm, suggestions: []});
-            } else {
-                trie.update(nm, {code: item.code, types: item.types, name: nm, suggestions: node.metaData.suggestions});
-            }
-        });
+        const defaultName = item.name;
```

Comment:
```suggestion
        const englishName = item.name;
```
To keep things consistent, I think we need to specify this as `englishName` (it's also clearer what we're doing that way).